src/main/java/net/countercraft/movecraft/combat/features/tracers/TNTTracers.java

```diff
@@ -25,8 +30,14 @@
 public class TNTTracers extends BukkitRunnable implements Listener {
     public static double TracerRateTicks = 5.0;
     public static long TracerMinDistanceSqrd = 360;
+    public static long TracerDelayTicks = 5;
+    public static long TracerExplosionDelayTicks = 5;
     public static Particle TracerParticle = null;
     public static Particle ExplosionParticle = null;
+    public static Material TracerBlock = null;
+    public static Material TracerExplosionBlock = null;
+    private static BlockData tracerBlockData;
+    private static BlockData tracerExplosionBlockData;
```

Comment:
For consistency these should use `PascalCase`.

Reply:
fixed issue